product-service/src/main/java/com/extremecoder/productservice/utils/UrlConstraint.java

```diff
@@ -0,0 +1,17 @@
+package com.extremecoder.productservice.utils;
+
+public final class UrlConstraint {
+
+    private UrlConstraint() {
+    }
+
+    private static final String API = "/api";
+    private static final String VERSION = "/v1";
```

Comment:
What if the version becomes `v2` few days back? 
- At that time we might have refactor this variable `VERSION` as `VERSION1`. And for v2 we might go for `VERSION2`. Instead the variable name can be `INITIAL_VERSION` or simply `VERSION1`.

Reply:
Changed it. Thank you :)

Reply:
You are welcome.